admin/src/pages/UserSearch.vue

```diff
@@ -70,11 +82,11 @@ export default {
   },
   methods: {
     unconfirmedRegisterMails() {
-      this.filterCheckedEmails = this.filterCheckedEmails ? null : true
+      this.filterByActivated = this.filterByActivated === null ? false : null
       this.getUsers()
     },
     deletedUserSearch() {
-      this.filterDeletedUser = this.filterDeletedUser ? null : true
+      this.filterByDeleted = this.filterByDeleted === null ? true : null
```

Comment:
Same as line 85
```suggestion
      this.filterByDeleted = !this.filterByDeleted ? true : false
```

Reply:
I like that we can see explizit that we switsch between `null` and `true`.
And then the logic works already if we include `false` in the future as well.